.github/workflows/ci.yml

```diff
@@ -11,24 +10,138 @@ on:
   workflow_dispatch:
 
 jobs:
-  test:
+  project-test:
     runs-on: ubuntu-latest
+    continue-on-error: true # allow jobs to complete even if some fail
     strategy:
       matrix:
-        java_version: [8]
+        java: [8, 11, 17, 21]
       max-parallel: 4
-    name: Test JDK ${{ matrix.java_version }}
+    name: JDK ${{ matrix.java }} | Project
     steps:
-      - name: Setup Maven Action
-        uses: s4u/setup-maven-action@v1.18.0
```

Comment:
Is this change necessary to testing on all LTS?

Reply:
More or less. Right now, `setup-maven-action` doesn’t really add value: it just avoids having to declare `actions/checkout` manually.